api/controllers/feedback-controller.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+var feedbacklib = require('../lib/feedback-lib');
+
+module.exports.addFeedbackForUser = function (req,res) {
+    var options = {
+        title: req.swagger.params.body.value.title,
+        feedbackBody: req.swagger.params.body.value.feedbackBody,
+        feedbackFor: req.swagger.params.body.value.feedbackFor,
+        userId: req.swagger.params.userId.value
+    };
+    feedbacklib.addFeedbackForUser(options, function(err, data) {
+        if(err)
+            res.json({'message': "Feedback for the user cannot be added"});
+        else
+            res.json(data);
+    });
+};
+
+module.exports.getFeedbacksForUser = function (req,res) {
+    var options = {
+        userId: req.swagger.params.userId.value
+    };
+    feedbacklib.getFeedbacksForUser(options, function(err, data) {
```

Comment:
if the returned data is empty, should we handle that here?

Reply:
if there are no feedback for the user, then the returned array should be empty.

in short : 200 OK and body : `[]`